### DISCUSSION TITLE
Image squash issue?

### DISCUSSION THREAD

**Discussion by shagerich** [praydog/REFramework#413] [General Questions] [answered]
I'm not sure if this is just how it's supposed to look but, in RE3 everything is squashed and stretched horizontally. Like fitting 4:3 to 16:9, but way worse. Everything, loading screens, the world in game, Main menu, everything. Not sure what's up. Are there setting that need changing? Installation looked straight forward enough. Everyone describes this as an S tier VR mod, and that is not the experience I'm having, haha. It's seems like something is up. Or, maybe this is how it's supposed to be? Any help? I've tried on my Rift S and my WMR, both are the same.

core i7 - 10700kf @ 3.80GHz 8 cores
32 GB RAM
GTX1080

**Comment by praydog** [comment]
Turn off borderless window mode. If that doesn't work, change your in-game resolutions around (the actual game's settings) until it looks normal. You can also download a nightly build here: https://github.com/praydog/REFramework-nightly/releases

**Reply by shagerich** [reply]
Ok, I'll try that! Thanks!

**Comment by shagerich** [comment, marked as answer]
ok, so I was sure I read the instructions, like, 8 times. I read them once more and there it was. delete openvr_api.dll. Now it works as intended and is awesome. Thank you praydog, for the quick response and the awesome mods. You are a gentleman and a scholar
